packages/mds-stream/index.ts

```diff
@@ -179,6 +207,38 @@ async function readStreamGroup(
       id || '>'
     ]
   )
+
+  if (results) {
+    const [result] = results
+    return result
+  }
+
+  return [stream, []]
+}
+
+async function getStreamInfo(stream: Stream) {
+  const client = await getClient()
+  try {
+    const [
+      ,
+      length,
+      ,
+      radixTreeKeys,
+      ,
+      radixTreeNodes,
+      ,
+      groups,
+      ,
+      lastGeneratedId,
+      ,
+      firstEntry,
+      ,
+      lastEntry
+    ] = await client.xinfoAsync('STREAM', stream)
+    return { length, radixTreeKeys, radixTreeNodes, groups, lastGeneratedId, firstEntry, lastEntry }
+  } catch (err) {
```

Comment:
We should probably at least log if an error occurs (unless an error occurring here isn't something to care about)

Reply:
It's not. It will only throw if the stream doesn't exist, which is possible the first time this thing runs.